src/Azure/AzureAD/test/FunctionalTests/Microsoft.AspNetCore.Authentication.AzureAD.FunctionalTests.csproj

```diff
@@ -15,6 +15,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Authorization" />
```

Comment:
These probably aren't needed now that we use indirectreferences.props. It's quite tedious to remove all of these but might be worth it.